### PR TITLE
new report, rate limits

### PR DESCRIPTION
**Pull request by toddocon**

### CODE REVIEW

**Comment by toddocon** [comment]
changed script name but forgot to change name in the help message. Updated that just now.

File: reports/rate-limit-violator.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        (-h|--help)     usage 2>&1;;
+        (--)            shift; break;;
+        (-*)            usage "$1: unknown option";;
+        (*) break;;
+    esac
+done
+
+execute << EOF
+    zgrep "rate_limit_remaining=0" /var/log/github/unicorn.$LOG |
+        grep -oP "rate_limit_key=\K[^ ]*" |
```
Comment (larsxschneider) [inline reports/rate-limit-violator.sh]: Very nice! Minor nit: I usually use single quotes for strings. That ensures nothing will be interpreted by the shell: https://www.gnu.org/software/bash/manual/html_node/Single-Quotes.html

**Comment by toddocon** [comment]
I had to merge in your new changes into my fork. Wonder if this pull request is all messed up now?

**Comment by larsxschneider** [comment]
No. Everything is perfect! I'll just squash merge to keep the history clean :)

**Pull request merged**